src/Symfony/Component/Serializer/Mapping/Loader/AnnotationLoader.php

```diff
@@ -243,7 +252,7 @@ private function isKnownAttribute(string $attributeName): bool
     /**
      * @return object[]
      */
-    private function getClassAnnotations(\ReflectionClass $reflector): array
+    private function getClassAttributes(\ReflectionClass $reflector): array
```

Comment:
Please revert. This method reads Doctrine annotations only. Thus, its name is accurate. On 7.0, the method has been removed already.

Reply:
I think I've confused it with the Validator's AnnotationLoader, which reads attributes. I reverted